### ISSUE TITLE
Create an Examples Folder

### ISSUE THREAD

**Issue by grantkemp** [s-yadav/radialIndicator#22]
Would be useful to have a simple hello world showing this library working without JQuery in an html page

**Comment by bala41288** [comment]
I agree with the above. An example of how to implement would be very helpful. We are lost after going to your site and documentation. It works well in the documentation but not when we try.

**Comment by LemonBaba** [comment]
We will be adding docs in the next few days
